agent/functional_tests/tests/functionaltests_test.go

```diff
@@ -758,6 +758,9 @@ func testContainerMetadataFile(t *testing.T, taskName, awslogsPrefix string) {
 	defer agent.Cleanup()
 	// TODO: Bump version to 1.24.0 (or next release after 1.23.0)
 	agent.RequireVersion(">=1.22.0")
+	if agent.getHostPublicIPv4AddressFromEC2Metadata() == "" {
```

Comment:
sorry that i missed this.. i don't think the TestAgent struct has this method?

Reply:
maybe just do something like this - https://github.com/aws/amazon-ecs-agent/blob/ebac2200d4952ff1bb08319fdabbd0e0eb1cebfb/agent/functional_tests/tests/functionaltests_test.go#L817-L819